### PR TITLE
test(longevity-large-partition): Adding a new large partition test with smaller dataset and shorter

### PR DESCRIPTION
**Pull request by roydahan**
…th smaller dataset and shorter

Adding a short longevity with large paritions dataset using scylla-bench.
Targeted to run about 8h.

### CODE REVIEW

**Comment by roydahan** [comment]
Still in testing, do not review.

**Comment by roydahan** [comment]
https://snapshot.raintank.io/dashboard/snapshot/DuOKGkxVyUWucg39l3qOxoaz5hndKvoY

The scylla-bench configured with iterations only, it has enough iterations to continue for 8h, the the duration will stop it.

File: test-cases/longevity/longevity-large-partition-8h.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=5555 -partition-offset=101 -clustering-row-size=uniform:1024..2048 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=30s",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=5555 -partition-offset=201 -clustering-row-size=uniform:1024..2048 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=30s",
+                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=100 -clustering-row-count=5555 -partition-offset=301 -clustering-row-size=uniform:1024..2048 -concurrency=10 -connection-count=10 -consistency-level=quorum -rows-per-request=10 -timeout=30s"
```
Comment (yarongilor) [inline test-cases/longevity/longevity-large-partition-8h.yaml]: i think a '-validate-data' should be added for above lines

File: test-cases/longevity/longevity-large-partition-8h.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=100   -partition-offset=426 -clustering-row-size=uniform:2048..5120   -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -iterations 10",
+             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=10    -partition-offset=451 -clustering-row-size=uniform:5120..8192   -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -iterations 10",
+             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=25 -clustering-row-count=1     -partition-offset=476 -clustering-row-size=uniform:8192..10240  -concurrency=10  -connection-count=10  -consistency-level=quorum -rows-per-request=10 -timeout=30s -iterations 10",
```
Comment (yarongilor) [inline test-cases/longevity/longevity-large-partition-8h.yaml]: i think a '-validate-data' flag should be added for above lines

File: test-cases/longevity/longevity-large-partition-8h.yaml (hunk 1 above)
Comment (juliayakovlev) [reply test-cases/longevity/longevity-large-partition-8h.yaml]: No, there is the problem in scylla-bench to support a few commands with validation. 
This is the [issue](https://github.com/scylladb/scylla-bench/issues/30)

File: test-cases/longevity/longevity-large-partition-8h.yaml (hunk 2 above)
Comment (juliayakovlev) [reply test-cases/longevity/longevity-large-partition-8h.yaml]: See my answer above

File: test-cases/longevity/longevity-large-partition-8h.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+#                  "scylla-bench -workload=uniform -mode=read -replication-factor=3 -partition-count=1000 -clustering-row-count=200000 -clustering-row-size=uniform:10..10240 -rows-per-request=10 -consistency-level=quorum -timeout=30s -partition-offset=1001 -concurrency=100 -connection-count=100 -duration=6420m"
+#                 ]
+
+n_db_nodes: 5
+n_loaders: 4
+n_monitor_nodes: 1
+
+round_robin: true
+
+instance_type_db: 'i3.2xlarge'
```
Comment (juliayakovlev) [inline test-cases/longevity/longevity-large-partition-8h.yaml]: Is 'i3.2xlarge' have enough space?

Comment (roydahan) [reply test-cases/longevity/longevity-large-partition-8h.yaml]: Yes, the dataset is not too big and tested as you can see above.

**Pull request merged**